advocacy_docs/edb-postgres-ai/console/estate/agent/install-agent.mdx

```diff
@@ -114,16 +114,14 @@ Entries under `databases` utilize the following format:
 
 ```yaml
 databases:
-    <database_name_1>:
-        dsn: "$DSN1"
-        tags:
-            - "<tag-one>"
-            - "<tag-two>"
-    <database_name_2>:
-        dsn: "$DSN2"
-        tags: 
-            - "<tag-one>"
-            - "<tag-two>"
+  - dsn: "$DSN1"
```

Comment:
As above - we have another PR where resource_id is used. The other option is that resource_id has been dropped, so in that case we should have a formal reference for the yaml in here.